src/Icons/Cart.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  fillColor?: string
+  fillOpacity?: string
+  stroke?: string
+  [x: string]: unknown // for the rest property
+}
+
+const Cart: React.FunctionComponent<Props> = ({ width = '37', height = '28', fillColor="white", fillOpacity="0.3", stroke="white", ...rest }) => (
+  <svg width={width} height={height} viewBox="0 0 37 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
xmlns="http://www.w3.org/2000/svg" isn't needed since you are inlining the SVG. That url is only when you are fetching the SVG over the network from another URL